scripts/init-database.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+import MySQLdb
+
+values = list()
+
+
+def retreiveValues():
+    stream = open("../.config/bassa/config.yml", "r")
+    docs = yaml.load_all(stream)
+    global values
+    for doc in docs:
```

Comment:
Why we collect all variables like this? Can't we access them only when needed, like `docs.database.database_name`

Anyway, `docs` is not a good name for this, shall we change it to `configs`?